extensions/cornerstone-dicom-rt/src/utils/promptHydrateRT.ts

```diff
@@ -15,23 +15,28 @@ function promptHydrateRT({
   hydrateRTDisplaySet,
 }) {
   const { uiViewportDialogService } = servicesManager.services;
-
+  const extensionManager = servicesManager._extensionManager;
+  const appConfig = extensionManager._appConfig;
   return new Promise(async function (resolve, reject) {
-    const promptResult = await _askHydrate(uiViewportDialogService, viewportId);
+    const promptResult = appConfig?.disableConfirmationPrompts
+      ? RESPONSE.HYDRATE_SEG
+      : await _askHydrate(uiViewportDialogService, viewportId);
 
     if (promptResult === RESPONSE.HYDRATE_SEG) {
       preHydrateCallbacks?.forEach(callback => {
         callback();
       });
 
-      const isHydrated = await hydrateRTDisplaySet({
-        rtDisplaySet,
-        viewportId,
-        toolGroupId,
-        servicesManager,
-      });
+      window.setTimeout(async () => {
```

Comment:
why?

Reply:
viewport.element is undefined otherwise, not sure what the cause is, guess it was never a problem before because the user would be too slow to click hydrate for that to happen.